python/setup.py

```diff
@@ -10,7 +10,7 @@
     if sys.version_info.major == 2:
         libraries = ["boost_python27-mt"]
     elif sys.version_info.major == 3:
-        libraries = ["boost_python36-mt"]
+        libraries = ["boost_python3"+str(sys.version_info.minor)+"-mt"]
```

Comment:
This isn't really my best code... :) Since every python version will have major, minor info, then the 'else' is never going to be called. 

You could probably remove this entire `if` statement and replace it with:

```
libraries = ["boost_python{major}{minor}-mt".format(major=sys.version_info.major, minor=sys.version_info.minor)]
```